packages/account/src/Sections/Verification/Helpers/verification.js

```diff
@@ -1,19 +1,25 @@
-export const populateVerificationStatus = account_status => {
-    const { attempts, document, identity, needs_verification } = account_status.authentication;
+const populateVerificationStatus = account_status => {
+    const { attempts, document, identity, needs_verification } = {
+        ...account_status.authentication,
+        attempts: account_status.authentication.attempts || {},
+        needs_verification: account_status.authentication.needs_verification || {},
+    };
```

Comment:
Can we add  Optional chaining operator(?.) here?

Reply:
done